### DISCUSSION TITLE
Help a noob

### DISCUSSION THREAD

**Discussion by Morcegolas** [bluenviron/mediamtx#2492] [Questions]
Good morning, I'm using a raspberry pi 3 with raspberry HQ camera. Already downloaded, extract and move to another folder Mediamtx. After that I created meadimtx.yml and pasted all from here and started with ./mediamtx and it starts listening on some ports. 
What is the address so I can view it on another computer? I already tried some addresses in my browser but without any luck. 
Hope someone can give me a help. 

Thanks.

**Comment by dillanspencer** [comment]
Hey man! If you just want to view the stream in the browser you can use the WebRTC port. You just need to find the ID of the path for that stream using the API. You can use `curl {MediaMTX ip}:9997/v2/paths/list` to get the list of stream and you will find the id of the stream there. Then in your browser go to `{MediaMTX ip}:8889/{cameraID}` and you will get the WebRTC stream for the camera. Make sure in the mediamtx.yml you have WebRTC enabled and that the WebRTC port is 8889

**Comment by aler9** [comment]
Hello, the README contains instructions on how to ingest a video stream from the Raspberry Pi camera to the server:
https://github.com/bluenviron/mediamtx#raspberry-pi-cameras

The README also contains instructions on how to read that stream with a web browser:
https://github.com/bluenviron/mediamtx#web-browsers-1

or VLC:
https://github.com/bluenviron/mediamtx#vlc

[a comment by an automated account is not shown]
